workspaces/variables.md

```diff
@@ -22,6 +22,11 @@ env | grep CODER_
         <th>Environment variable</th>
         <th>Description</th>
     </tr>
+    <tr>
+        <td><code>CODER_AGENT_TOKEN</code></td>
```

Comment:
can we add a separate section, or otherwise label that this is not covered by our backward compatibility guarantees and may be changed or removed in a later release? This is really an internal implementation detail, which is why we didn't document it in the first place, but I understand that people may want to understand what these variables are for